### DISCUSSION TITLE
pip-requirements

### DISCUSSION THREAD

**Discussion by Legend94rz** [PaddlePaddle/PaddleSpatial#127] [BaiduKDDCup2022] [answered]
Could you please install these python package to the pytorch env?
```
Torchfast
transformers
numba
```

**Comment by ramber1836** [comment]
Could you please provide example pip requirements for these pkgs?

**Reply by Legend94rz** [reply]
no version limitations, just as above.

**Comment by ramber1836** [comment, marked as answer]
Already added in the Pytorch environment.
